hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/key/KeyUtils.java

```diff
@@ -33,6 +33,10 @@ public class KeyUtils {
 
     public static final Key IMMUTABILITY_SENTINEL_KEY =
             Key.newBuilder().keyList(KeyList.DEFAULT).build();
+    public static final Key ALL_ZEROS_INVALID_KEY = Key.newBuilder()
+            .ecdsaSecp256k1(Bytes.fromHex("0000000000000000000000000000000000000000"))
```

Comment:
A compressed ECDSA key [must begin ](https://crypto.stackexchange.com/questions/108091/how-to-determine-the-prefix-of-a-secp256k1-compressed-public-key) with `0x02` or `0x03`; so this key isn't even in principle valid. 

An all-zero ED25519 key, however, **is** valid in principle; but not in practice, since there's no way to find the corresponding private key.

I suggest we use an all-zero ED25519 key here; or make this a valid ECDSA key by prefixing `0x02`.

Reply:
Agreed, ecdsa builder was used by mistake. Switched to all-zero ed25519 curve key